applications/graphQL/tests/test_query.py

```
@@ -0,0 +1,121 @@
+# TODO: Refactor
```

Pylint (3.3.7)

[warning] Unknown option value for '--disable', expected a valid pylint message and got 'long-suffix'

(W0012)